src/main/java/ubc/pavlab/rdp/controllers/UserController.java

```diff
@@ -80,6 +80,7 @@ public ModelAndView model( @PathVariable Integer taxonId ) {
         } else {
             modelAndView.addObject( "viewOnly", null );
             modelAndView.addObject( "user", user );
+            modelAndView.addObject( "userGenes", user.getGenesByTaxonAndTier( taxon, TierType.MANUAL ) );
```

Comment:
Was this added just to avoid queries in view? If so, we should not include the change and instead scheduled it later with a proper fix for #172.

Reply:
It's actually needed since we cannot instantiate `TierType.MANUAL` in the view.